src/components/Layout/TopBackground.tsx

```diff
@@ -30,6 +30,7 @@ export const TopBackground: FunctionComponent<TopBackground> = ({
         style,
       ]}
       colors={[primaryColor, secondaryColor]}
+      //colorsv2={["#303D67", secondaryColor]}
```

Comment:
haha small nit here, maybe let's remove this change here? We can always add the change later on 😊 

So only `colors.tsx` will have changes for this PR (:

Reply:
have removed.